extendr-api/src/robj.rs

```diff
@@ -318,13 +318,21 @@ impl Robj {
 
     /// Get an iterator over a string vector.
```

Comment:
Maybe add here in the documentation that it also works for factors?

Reply:
Added...